observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,61 @@ These release notes track the development of the Network Observability Operator
+==== eBPF performance enhancements
+*NetObserv / Health* dashboard integration with eBPF metrics allows you to view alerts in two causes: when the Network Observability eBPF agent hashmap table is full, which cases the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered. For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
+
+[id="network-observability-cli-1.6"]
+==== Network Observability CLI
+The Network Observability CLI allows you to debug and troubleshoot with network observability without needing to install the Network Observability Operator. You can capture and visualize flow and packet data in real-time with no persistent storage requirement during the capture. For more information, see <CLI_LINK_HERE_AFTER_FEATURE_PR_MERGE>
+
+[id="network-observability-no-show-duplicates-1.6"]
+==== Duplicates no longer shown
+Previously, duplicated flows were stored as individual flows, and flagged as duplicate if there were multiple of the same kind of flows. You could choose to show the duplicated flows using 
+the *Duplicate flows* Query Option. Now instead of having multiple, individual flows, there is one flow with a list of matching *Interfaces* and *Interface directions*. Since the duplicated flows are merged in a list to a single flow data, Loki storage demand decreases significantly. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-quickfilternw-observe-network-traffic[Quick filters].
+
+[NOTE]
+===
+With this update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any saved queries usi ng these fields need to be updated. 
+===
+
+[id="network-observability-lokiless-enhancements-1.6"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now do more than ever with the Network Observability Operator without relying on Loki for storage. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-ebpf-collection-filtering-1.6"]
+==== eBPF collection rule-based filtering
+You can use rule-based filtering to control the volume of packets cached in the eBPF flow table. When this option is enabled, the* Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view as well as additional query options in *Observe* -> *Metrics*. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
```

Comment:
```suggestion
You can use rule-based filtering to reduce the volume of created flows. When this option is enabled, the *Netobserv / Health* dashboard for eBPF agent statistics has the *Filtered flows rate* view. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
```

I would not mention "cached in the eBPF table" here, users may not really understand that, but rather would tell about the consequences (reducing number of flows). Also not sure what is "as well as additional query options in *Observe* -> *Metrics*" (we don't directly change anything from that page; the intent might be to tell that a new metric is accessible from there, but that's said implicitely already when saying that the dashboard has a new chart for filtered flows